spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Cell.scala

```diff
@@ -80,7 +80,9 @@ abstract class Cell[T : ClassTag](
    * Please refer to SimpleRNN or LSTM for reference.
    * @return
    */
-  def preTopology: AbstractModule[Activity, Activity, T] = null
+  var preTopology: TensorModule[T]
```

Comment:
Do we need to handle the new variable in model serialization? @wzhongyuan

Reply:
yes, we need to handle it, otherwise the behaviour will be different after deserialization for a 'true' value